storio-sqlite/src/main/java/com/pushtorefresh/storio/sqlite/Changes.java

```diff
@@ -47,8 +73,23 @@ public static Changes newInstance(@NonNull Set<String> affectedTables) {
      */
     @NonNull
     public static Changes newInstance(@NonNull String affectedTable) {
+        return newInstance(affectedTable, null);
+    }
+
+    /**
+     * Creates {@link Changes} container with info about changes.
+     *
+     * @param affectedTable table that was affected.
+     * @param affectedTag tag that was affected.
+     * @return new immutable instance of {@link Changes}.
+     */
+    @NonNull
+    public static Changes newInstance(@NonNull String affectedTable, @Nullable String affectedTag) {
         checkNotNull(affectedTable, "Please specify affected table");
-        return new Changes(Collections.singleton(affectedTable));
+        final Set<String> tags = affectedTag == null
+                ? Collections.<String>emptySet()
+                : singleton(affectedTag);
```

Comment:
It makes sense to reuse static import for `singleton` next line or remove it completely.